buildapp/buildapp.py

```diff
@@ -118,4 +120,6 @@ def main():
     print('buildapp completed successfully!')
 
 if __name__ == "__main__":
+    if not isdir(RELTOOLS_ROOT):
```

Comment:
Do not use `os` module for that, but the `mkdir` method of the `pathlib.Path` class (e.g. `RELTOOLS_ROOT.mkdir(exist_ok=True)`, which also removes the need of the `isdir` check)

In addition, I'd like u to write here why'd u think u should create that empty directory, instead of running `buildapp_fetch_tools` as noted in the requirements